.travis.yml

```diff
@@ -10,11 +10,13 @@ env:
     - SKIP_WRITE_GIT_CHANGELOG=1
   matrix:
     - WITH_SCIPY=true WITH_CODECOV=true
-install:
+before_install:
   - pip install --upgrade setuptools pip
+install:
   - if [[ ${WITH_SCIPY} == true ]]; then pip install scipy; fi
-  - pip install --upgrade -r requirements-tests.txt
   - pip install --upgrade .
+before_script:
```

Comment:
Why do you need these steps to be in `before_install` and `before_script`? Why not just in `install`?

Reply:
Need is a large word I think. I expect it to make deploy a little faster since it won't have to install all our testing equipment.

Reply:
Good idea.